lib/index.js

```diff
@@ -178,15 +133,19 @@ module.exports = {
           packagesFolder);
           fs.readdirSync(packagesFolder)
             .filter(function (name) {
-              return name.slice(0, 7).toLowerCase() !== 'system.'
+              return name.slice(0, 7).toLowerCase() !== 'system.';
             })
             .map(function (folderName) {
               return Dependecy.from.folderName(folderName);
             })
             .forEach(function (dep) {
-              injectPath(dep);
-              flattendPackageList[dep.name] =
-                flattendPackageList[dep.name] || dep;
+              injectPath(dep, packagesFolder);
+              // only add a package from packages folder if version is different
+              if (flattendPackageList[dep.name] &&
+                flattendPackageList[dep.name].version !== dep.version) {
+                // prefer found from packages folder (dep) over existing
+                flattendPackageList[dep.name] = dep;
+              }
```

Comment:
Let's add a `debug` message in this `if` clause to show which dep is involved, what version was extracted from the manifest file, and what version is overwriting it from the `packages` folder.

Reply:
updated.
```
snyk For package jQuery the version 1.1.1was extracted from manifest file.
snyk We are overwriting it with version 3.2.1 from the packages folder
```